plugin/src/tryGetObjects.spec.lua

```diff
@@ -0,0 +1,39 @@
+-- TODO: Have some documentation/script in order to force the rbxassets to be there
```

Comment:
Does the API work with `rbxassetid://` URLs? What about a model that we know exists in the Studio distribution? We should rely on those if we can.

Reply:
Looks like it does!

Reply:
As for using something in the Studio distribution, it's possible if we use a multi-rooted result instead of the current one.

Reply:
Though I'm a bit worried about relying on anything internal to Studio, honestly.